webpack.common.js

```diff
@@ -16,46 +15,46 @@ module.exports = {
         'sva-finder': './src/Components/SvaFinder.js',
         'act-widget': './src/Components/ActWidget.js',
         'donation-widget': './src/Components/DonationWidget.js',
-        pdfworker: './src/Utility/PdfWorker.js',
         'test-interface': './src/test-interface.js',
         // We need to define a dummy entrypoint that requires all our translation files, otherwise Webpack will not
         // process them.
         'translations-dummy': ['./src/i18n/de.json', './src/i18n/en.json', './src/i18n/fr.json', './src/i18n/pt.json'],
     },
     optimization: {
-        minimize: true,
-        minimizer: [
-            new TerserPlugin({
-                sourceMap: true,
-                extractComments: false,
-                cache: true,
-                parallel: true,
-                terserOptions: {
-                    mangle: {
-                        reserved: [
-                            'ActionButtonPlaceholder',
-                            'NewRequestButtonPlaceholder',
-                            'CompanySelectorPlaceholder',
-                            'RequestFormPlaceholder',
-                            'DynamicInputContainerPlaceholder',
-                            'SignatureInputPlaceholder',
-                            'RequestTypeChooserPlaceholder',
-                            'RecipientInputPlaceholder',
-                            'TransportMediumChooserPlaceholder',
-                        ],
-                    },
+        splitChunks: {
+            cacheGroups: {
+                vendors: {
+                    name: 'vendors',
+                    chunks: 'all',
+                    test: /[/\\]node_modules[/\\]/,
+                    minChunks: 2,
                 },
-            }),
-        ],
+                commons: {
+                    name: 'commons',
+                    chunks: 'initial',
+                    minChunks: 2,
+                },
+            },
+        },
     },
     output: {
         filename: 'js/[name].gen.js',
         chunkFilename: 'js/[name].bundle.gen.js',
         publicPath: '/',
         path: path.resolve(__dirname, 'static'),
+        globalObject: 'self',
     },
     module: {
         rules: [
+            {
+                test: /\.worker\.js$/,
+                use: [
+                    { loader: 'worker-loader' },
```

Comment:
Is there any benefit to using this loader? We are currently thinking about switching our JS build from Webpack to Hugo with esbuild, so I would like to avoid adding Webpack-specific features that we later need to remove again/rebuild. :D

Reply:
If we *do* keep that, the file names need to be adjusted. The currently used `pdf.worker.gen.worker.js` is not covered by our `.gitignore`.

Reply:
I suppose I can get rid of it, fixed an issue I was having with split chunks and makes the worker a bit nicer to work with (automatically adds the entry, nice loading by importing)